src/DotNetWorker.Grpc/GrpcRetryContext.cs

```diff
@@ -14,8 +14,8 @@ public GrpcRetryContext(RetryContextMessage retryContext)
             _retryContext = retryContext;
         }
 
-        public override int RetryCount => _retryContext.RetryCount;
+        public override int? RetryCount => (_retryContext != null)?_retryContext.RetryCount: null;
```

Comment:
nit: parenthesis is not necessary, also spacing consistency

```suggestion
        public override int? RetryCount => _retryContext != null ?_retryContext.RetryCount : null;
```